Jobsway2goMvc/Controllers/CollectionsController.cs

```diff
@@ -204,8 +204,15 @@ public async Task<IActionResult> DeleteConfirmed(int id)
                 return Problem("Entity set 'ApplicationDbContext.Collections'  is null.");
             }
             var collection = await _context.Collections.FindAsync(id);
+            
             if (collection != null)
             {
+                if (collection.Job.Count() > 0)
+                {
```

Comment:
`Job.Count` will not work properly because it is commented in the model. Remove from comment and also rename it to Jobs, and then check. Test before commiting.

Reply:
fixed this